tests/plugins/toolsets/test_internet.py

```diff
@@ -112,7 +112,12 @@ def test_html_to_markdown(fixture: Fixture):
     assert match, f"Values mismatch. Run the following command to compare expected with actual: `diff {fixture.expected_output_file_path} {actual_file_path_for_debugging}`"
 
 
-def test_fetch_webpage():
+def test_fetch_webpage(responses):
+    responses.get(
```

Comment:
whats this added for? testing web connectivity?

Reply:
Added pytest-responses it is patching all requests by default and raise an error.
e.g. this test was actually doing real network call so i patched it. No reason for tests to do actual call unless there is a real reason behing it.